client/components/Resources/Document/MediaList.js

```diff
@@ -9,28 +9,30 @@ import InvalidPage from '../../common/InvalidPage';
 const MediaList = (props) => {
   const { documents, isMedia, loading, instanceName, addMediaUrl } = props;
   return (
-    <Grid columns={4} className="ui loading">
-      <Grid.Row className="animated fadeIn">
-        {documents.results.map((item) => {
-          return isMedia ? (
-            <MediaCard item={item} key={item._id} {...props} />
-          ) : (
-            <DocumentCard item={item} key={item._id} {...props} />
-          );
-        })}
-      </Grid.Row>
-      {!loading && documents.results.length < 1 && (
-        <Grid.Row className="ui loading center aligned animated fadeIn">
-          <InvalidPage
-            pathLabel={`Add a ${instanceName}`}
-            errorMessage={`No ${instanceName} to display`}
-            errorDescription={`Please add ${instanceName}`}
-            path={addMediaUrl || '/'}
-          />
+    <React.Fragment>
+      <Grid columns={4} className="ui loading">
```

Comment:
Nice job @eduhmik I tested it and it works fine.
```suggestion
     <React.Fragment>
       <Grid columns={4} className="ui loading">
```
`<React.Fragment> ` can be removed here.

Reply:
Hey @eltNEG, removed <React.Fragment>. When actually do we need to use this?

Reply:
An example:
```
const Example = (
               <div>
                       <Element1 />
                       <Element2 />
               <div/>
```
If the outter `div` in the above is not doing anything apart from wrapping components `Element1` and `Element2`, then, the div can be replaced with `Fragment`. 

`Fragment` prevents the addition of an unnecessary element to the dom-tree.

You can get more details [here](https://reactjs.org/docs/fragments.html)